### ISSUE TITLE
Install in Safari

### ISSUE THREAD

**Issue by kczempinski** [DavidBuchanan314/Turbo-Recadmiumator#6]
Any possibility to support Safari on Mac OS?

**Comment by DavidBuchanan314** [comment]
I don't have any macos systems in my daily rotation so I can't commit to maintaining safari support, but PRs would be accepted

**Comment by DavidBuchanan314** [comment]
But, I would question the need for it? Doesn't Safari get you 4K by default?

**Comment by kczempinski** [comment]
Yes, 4k works by default, but surround sound does not. I used Boom3D for Netflix ( https://apps.apple.com/pl/app/boom3d-5-1-audio-for-netflix/id6445882848?l=pl&mt=12 ) but now I receive the same error (UI3013)

**Comment by kczempinski** [comment]
I found a solution to temporarily load the script into Safari:

1. Enable developer options in Safari<br>
2. Edit line 31 in cadmium-playercore-shim.js as below
`var cadmium_url = "https://assets.nflxext.com/en_us/ffe/player/html/cadmium-playercore-6.0044.268.911.js"`
3. Check _**Disable cross-origin restrictions**_ in developer settings 
<img width="948" src="https://github.com/DavidBuchanan314/Turbo-Recadmiumator/assets/3290514/fde75919-b76d-43e7-bcc5-18cf387a7338">
4. Play any video on Netflix<br>
5. From the menu click _**Develop**_ -> _**Show Page Source**_<br>
6. Find cadmium script in Sources<br>
<img width="300" src="https://github.com/DavidBuchanan314/Turbo-Recadmiumator/assets/3290514/9b737030-f640-4c2c-9979-c122489ed8ba"><br>
7. Right-click the script and select _**Create Response Local Override**_<br>
8. Click **_Import_** or _**Map to File**_ and paste the content of the modified _cadmium-playercore-shim.js_ or point to the file<br>
9. Repeat steps 3-8 after every Safari restart
